pkg/apis/kops/componentconfig.go

```diff
@@ -234,6 +228,13 @@ type KubeletConfigSpec struct {
 	// ExperimentalAllocatableIgnoreEviction enables ignoring Hard Eviction Thresholds while calculating Node Allocatable
 	ExperimentalAllocatableIgnoreEviction *bool `json:"experimentalAllocatableIgnoreEviction,omitempty" flag:"experimental-allocatable-ignore-eviction"`
 
+	// imageMinimumGCAge is the minimum age for an unused image before it is garbage collected. Default: "2m"
+	ImageMinimumGCAge *string `json:"imageMinimumGCAge,omitempty"`
```

Comment:
Should be backwards compatible.
```suggestion
	ImageMinimumGCAge *metav1.Duration `json:"imageMinimumGCAge,omitempty"`
```

Reply:
This requires changes to v1alpha2 and v1alpha3 , please review my latest pull request.